includes/admin/class-wcmp-admin.php

```diff
@@ -83,8 +83,49 @@ public function __construct()
         add_action("manage_shop_order_posts_custom_column", [$this, "addBarcodeToOrderColumn"], 10, 2);
 
         add_action( 'woocommerce_payment_complete', [$this, 'automaticExportOrder'], 1000);
+
+        add_action("init", [$this, "registerDeliveredPostStatus"], 10, 1);
+        add_filter("wc_order_statuses", [$this, "displayDeliveredPostStatus"], 10, 2);
+    }
+
+    /**
+     * Add delivered post type to order statuses list
+     */
+    public function registerDeliveredPostStatus(): void
+    {
+        register_post_status('wc-custom-delivered',
+            [
+                'label'                     => 'Delivered',
+                'public'                    => true,
+                'exclude_from_search'       => false,
+                'show_in_admin_all_list'    => true,
+                'show_in_admin_status_list' => true,
+                'label_count'               => _n_noop('Delivered (%s)', 'Delivered (%s)'),
+            ]
+        );
+    }
+
+    /**
+     * @param $order_statuses
+     *
+     * @return array
+     */
+    public function displayDeliveredPostStatus( $order_statuses ): array
```

Comment:
wat voor type is $order_statuses? en zodra je dat weet kan je het erbij zetten bijvoorbeeld `displayDeliveredPostStatus(string $order_statuses ): array`

Reply:
( $order_statuses ) spaties te veel?

Reply:
Yes even kijken